source/assets/images/core-016-a.svg

```diff
@@ -0,0 +1 @@
l0,-3.611l-2.423,-7.457l-0.557,0.254l2.28,8.714l0.7,2.1ZM113.087,38.638c-0.197,0.075 -0.392,0.152 -0.586,0.232l3.782,11.641l1.763,2.426l-0.346,-1.037l-4.613,-13.262ZM53.299,43.048l-6.948,9.587l9.606,-6.928c-0.818,-0.951 -1.707,-1.84 -2.658,-2.659ZM20.642,43.099c-0.918,0.795 -1.777,1.656 -2.571,2.575l9.41,6.837l-6.839,-9.412ZM138.336,43.08l-6.787,9.411l9.408,-6.784c-0.808,-0.939 -1.684,-1.817 -2.621,-2.627ZM105.568,43.164c-0.891,0.777 -1.725,1.616 -2.498,2.511l9.156,6.653l-6.658,-9.164ZM42.983,50.528l-0.092,0.279l0.698,-1.116l-0.606,0.837ZM27.317,38.947c-1.124,0.473 -2.206,1.025 -3.239,1.651l6.829,9.399l-3.59,-11.05ZM46.034,41.26l-2.846,8.643l1.963,-2.707l3.013,-4.817l-0.723,-0.482c-0.133,0.066 -0.283,0.103 -0.441,0.103c-0.462,0 -0.851,-0.314 -0.966,-0.74ZM112.223,38.986c-1.089,0.464 -2.139,1.003 -3.143,1.611l6.633,9.129l-3.49,-10.74ZM131.843,39.015l-3.475,10.598l6.517,-9.038c-0.973,-0.586 -1.989,-1.108 -3.042,-1.56ZM39.1,37.087l0,11.818l3.697,-11.229c-1.201,-0.285 -2.436,-0.484 -3.697,-0.589ZM124.1,37.087l0,11.754l3.664,-11.172c-1.191,-0.282 -2.414,-0.478 -3.664,-0.582ZM118.825,41.718l-1.073,0.488l2.148,6.612l0,-7.052c-0.147,0.085 -0.318,0.134 -0.5,0.134c-0.214,0 -0.413,-0.067 -0.575,-0.182ZM34.9,37.087c-1.235,0.103 -2.446,0.296 -3.624,0.572l3.624,11.154l0,-11.726ZM115.982,37.73c-0.173,0.043 -0.346,0.088 -0.518,0.134l0.836,2.536c0.1,0.1 0.1,0.1 0.1,0.2l0.263,1.004l0.503,-0.23l-1.184,-3.644ZM48.649,39.877l-0.651,1.058c0.001,0.022 0.002,0.043 0.002,0.065c0,0.022 -0.001,0.045 -0.002,0.067l0.697,0.464l0.758,-1.21c-0.265,-0.153 -0.533,-0.301 -0.804,-0.444ZM117.133,37.474c-0.288,0.057 -0.574,0.119 -0.858,0.186l1.166,3.589l0.963,-0.438c0.005,-0.061 0.016,-0.12 0.032,-0.178l-1.303,-3.159ZM46.782,38.988l-0.405,1.23c0.171,-0.136 0.388,-0.218 0.623,-0.218c0.307,0 0.583,0.139 0.766,0.358l0.437,-0.711c-0.466,-0.234 -0.94,-0.453 -1.421,-0.659ZM119.9,37.087c-0.765,0.064 -1.521,0.162 -2.265,0.293l1.132,2.746c0.173,-0.141 0.393,-0.226 0.633,-0.226c0.182,0 0.353,0.049 0.5,0.134l0,-2.947ZM51.574,36.93l2.772,-4.43l-0.824,-0.55l-2.776,4.515c0.279,0.151 0.555,0.306 0.828,0.465ZM115.583,33.714l-1.164,-2.823l-1.11,0.506c0.034,0.064 0.064,0.132 0.091,0.203l0.811,2.461c0.453,-0.127 0.91,-0.242 1.372,-0.347ZM108.409,18.235l0.509,1.537l0.626,-0.209l0.885,2.657l-0.629,0.21l2.726,8.225l1.641,-0.748l-4.267,-10.307c-0.415,-0.947 -0.794,-1.299 -1.491,-1.365ZM106.3,18.358l-46.985,6.687l0.185,0.555c0,0.2 0.3,0.3 0.5,0.3l46,-6.6c0.3,-0.1 0.5,-0.4 0.4,-0.7l-0.062,-0.2l-0.038,0l0,-0.042ZM109,10l-1,0l0.3,0.3c0.3,0.3 0.3,0.7 0.2,1c-0.1,0.3 -0.4,0.5 -0.8,0.5l-1.6,0l0.428,1.369l0.616,-0.206l0.885,2.657l-0.666,0.222l0.171,0.547l1.166,-0.389l0.3,0.9l-0.918,0.348l0.259,0.782c0.61,0.031 1.042,0.223 1.422,0.722c-0.591,-1.78 -0.775,-3.021 -0.263,-3.852c0.5,-0.7 1.6,-1.1 3.5,-1.1c1.3,0 1.9,-0.7 1.9,-1.4c0,-0.7 -0.6,-1.4 -1.9,-1.4c-1.169,0 -2,-0.169 -2.522,-0.478l-0.478,0.478l-1,-1ZM57,12.5l1.333,-0.5l-2.133,0c-0.1,0 -0.3,0 -0.4,0.1l-0.357,0.121l0.557,0.279l1,0ZM107,9" style="fill:#fff;"/></svg>
```

Comment:
Is this simply a visual element? Can we add the proper `role` and ARIA attributes, and give it a proper `title`?